desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -1227,10 +1227,10 @@ void MainTab::saveTaggedStrings(const QVector<dive *> &selectedDives)
 
 	if (diffTaggedStrings(cd->buddy, displayed_dive.buddy, addedList, removedList))
 		MODIFY_DIVES(selectedDives,
-			QStringList oldList = QString(mydive->buddy).split(QRegExp("\\s*,\\s*"), QString::SkipEmptyParts);
+			const QStringList oldList = QString(mydive->buddy).split(QRegExp("\\s*,\\s*"), QString::SkipEmptyParts);
```

Comment:
Please drop this commit - the code will be removed with the undo changes.

Reply:
np

Reply:
removed.